code-challenges.test.js

```diff
@@ -14,39 +14,150 @@
 
 // a) Create a test with expect statements using the variables provided.
 
-const secretCodeWord1 = "Lackadaisical"
-// Expected output: "L4ck4d41s1c4l"
-const secretCodeWord2 = "Gobbledygook"
-// Expected output: "G0bbl3dyg00k"
-const secretCodeWord3 = "Eccentric"
-// Expected output: "3cc3ntr1c"
+// Good fail: ReferenceError: codeMessage is not defined
+// Other fail: TypeError: codeMessage(...).toEqual is not a function
+test ('A function that takes in a string and returns a string with a coded message. The coded message converts "a" to 4, "e" to 3, "i" to 1, and "o" to 0.', () => {
```

Comment:
I don't see a describe block.  Make sure to set up your test using both the describe block and it(or test) block for a complete test.